client/src/panels/panel_declarations/finances/drr_dp_resources/crso_by_prog.js

```diff
@@ -144,10 +167,29 @@ class PlannedProgramResources extends React.Component {
           <div className="col-12 col-lg-8">
             <WrappedNivoBar
               data={data_by_year}
+              padding={0.3}
               keys={Object.keys(graph_data)}
               indexBy="year"
               colors={(d) => colors(d.id)}
               is_money={!is_fte}
+              {...{
+                ...(gap_year && {
+                  markers: [
+                    {
+                      axis: "x",
+                      value: gap_year,
+                      lineStyle: {
+                        stroke: tertiaryColor,
+                        transform: `translate(${
+                          0.3 * window_width_last_updated_at * 0.08 // Prob need media query to better handle
```

Comment:
Ehh doesn't really work all that well, I can try putting in media query to adjust better for 992px and smaller

Reply:
But maybe could just remove this line